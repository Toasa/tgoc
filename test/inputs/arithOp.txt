5 + 3;
5 - 3
5 * 3;
15 / 3
15 % 3