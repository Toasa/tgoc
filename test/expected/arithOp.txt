8
2
15
5
5
